app/src/routes/routesMap.js

```diff
@@ -128,7 +128,10 @@ const routesMap = {
   [OAUTH_SUCCESS]: '/authSuccess',
   [NOT_FOUND]: '/notfound',
 
-  ALL_ORGANIZATIONS_PAGE: redirectRoute('/', () => ({ type: ORGANIZATIONS_PAGE })),
+  ALL_ORGANIZATIONS_PAGE_INSTANCE_LEVEL: redirectRoute('/', () => ({ type: ORGANIZATIONS_PAGE })),
+  ALL_ORGANIZATIONS_PAGE: redirectRoute('/allOrganizations', () => ({
+    type: ORGANIZATIONS_PAGE,
+  })),
```

Comment:
Do we really need three organization pages? (ALL_ORGANIZATIONS_PAGE_INSTANCE_LEVEL, ALL_ORGANIZATIONS_PAGE, ORGANIZATIONS_PAGE)

Reply:
We need three organization pages or there will be other bugs